Sources/MapboxMaps/Gestures/GestureManager+GestureHandlerDelegate.swift

```diff
@@ -145,10 +145,10 @@ extension GestureManager: GestureHandlerDelegate {
 
         // Avoid contention with in-progress gestures
         // let toleranceForSnappingToNorth: CGFloat = 7.0
-        if mapView.cameraView.bearing != 0.0
+        if mapView.bearing != 0.0
             && pinchState != .began
             && pinchState != .changed {
-            if mapView.cameraView.bearing != 0.0 && isRotationAllowed() == false {
+            if mapView.bearing != 0.0 && isRotationAllowed() == false {
```

Comment:
I know this isn't new with PR, but it'd be more Swifty to write

```suggestion
            if mapView.bearing != 0.0, !isRotationAllowed() {
```